addons/sys_rack/stringtable.xml

```diff
@@ -3,27 +3,51 @@
     <Package name="sys_rack">
         <Key ID="STR_ACRE_sys_rack_mountRadio">
             <English>Mount Radio</English>
+            <German>Funkgerät einbauen</German>
         </Key>
         <Key ID="STR_ACRE_sys_rack_unmountable">
             <English>Unmountable (Locked)</English>
+            <German>Funkgerät fest verbaut</German>
         </Key>
         <Key ID="STR_ACRE_sys_radio_noGuiExternal">
             <English>Radio used externally. Only owner may configure.</English>
+            <German>Das Funkgerät wird extern benutzt. Nur der Besitzer kann es einstellen.</German>
         </Key>
         <Key ID="STR_ACRE_sys_rack_unmountRadio">
             <English>Unmount Radio (%1)</English>
+            <German>Entferne Funkgerät (%1)</German>
```

Comment:
I thinks this means using tools. But do you have to use tools to remove  for example the 152?